### PR TITLE
Enhance Kotlin guide

### PR DESCRIPTION
**Pull request by heubeck**
Tries to satisfy https://github.com/quarkusio/quarkus/issues/21297

### CODE REVIEW

File: docs/src/main/asciidoc/kotlin.adoc [hunk 1]
```diff
@@ -378,24 +377,52 @@ data class ResultItem(
 ...
 ----
 
+== Kotlin and the Kubernetes-Client
```
Comment (gsmet) [inline docs/src/main/asciidoc/kotlin.adoc:380]:
```suggestion
== Kotlin and the Kubernetes Client
```

File: docs/src/main/asciidoc/kotlin.adoc [hunk 2]
```diff
@@ -378,24 +377,52 @@ data class ResultItem(
...
+
+When working with the `quarkus-kubernetes` extension and have Kotlin classes bound to CustomResource definitions (like you do for building operators), you need to be aware, that the underlying fabric8 kubernetes-client uses its own static Jackson `ObjectMapper` s, which can be configured as follows with the `KotlinModule`:
```
Comment (gsmet) [inline docs/src/main/asciidoc/kotlin.adoc:382]:
```suggestion
When working with the `quarkus-kubernetes` extension and have Kotlin classes bound to CustomResource definitions (like you do for building operators), you need to be aware that the underlying Fabric8 Kubernetes Client uses its own static Jackson `ObjectMapper` s, which can be configured as follows with the `KotlinModule`:
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Nice stuff, thanks. I just spotted a couple minor things. Could you adjust and squash?

Thanks!

**Comment by heubeck** [comment]
> Nice stuff, thanks. I just spotted a couple minor things. Could you adjust and squash?
> 
> Thanks!

Did it.

We have to thank all of you ;)

**Review by gsmet — APPROVED** [review]

**Comment by heubeck** [comment]
Awesome @gsmet, you made my day... this was :smiling_imp: PR 666 :smiling_imp: in my GitHub life:
image

**Pull request merged**